### ISSUE TITLE
unix `ctime` is "change time" not "creation time"

### ISSUE THREAD

**Issue by zcutlip** [dhruvkb/pls#17]
## Description
In documentation as well as output, when `pls` refers to `ctime`, it characterizes that time as the file's creation time. This is a common misconception. On most unix and unix-like operating systems, `ctime` is The inode's "change time." In fact I believe there is no POSIX standard to track file creation time (sometimes called birthtime).

Some filesystems (e.g., APFS & HFS+ on macOS) do report file creation time, where as others (e.g., ext4 on Linux) do not.

How does that differ from `mtime` you may ask. `mtime` (as well as `ctime`) is updated when the *contents* of the inode are changed. If only the file's metadata (e.g., ownership, permissions, etc.) are changed, then only the inode's `ctime` will be updated.

## Reproduction
1. create a file: e.g., `ls -l > listing.txt`
2. `stat` it
3.  `pls -d ctime -d mtime -d atime` the file & see the respective times match
4. `chmod +x` the file
5. `stat` and `pls` the file as before
6. see that only `ctime` has been updated, which is clearly not the file's creation time

## Expectation
In documentation as well as output `ctime` should be referred to as "change time" or simply `ctime`

## Environment

maOS:

```
ProductName:	macOS
ProductVersion:	12.3.1
BuildVersion:	21E258
```

Ubuntu:

```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.4 LTS"
```

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

**Comment by dhruvkb** [comment]
I suppose the main source of confusion for me was that Windows stores the creation time as `ctime`. I'll fix it right away.

**Comment by dhruvkb** [comment]
I just deploy v4.0.1 to PyPI which includes commit 7623120 with the fix. It also adds a `btime` field (limited to macOS). The correction to the docs has also been published with commit 535de79.

**Comment by zcutlip** [comment]
Oh wow, that was quick work, nicely done and thank you!

Mostly for the sake of posterity when future people who stumble across this GH issue. I was wrong about EXT4 not tracking file creation time. It appears several filesystems supported by the Linux kernel do track file creation times, though working with those times is somewhat complicated and not standardized:
https://unix.stackexchange.com/questions/7562/what-file-systems-on-linux-store-the-creation-time#40093
https://unix.stackexchange.com/questions/50177/birth-is-empty-on-ext4/407305#407305
